### PR TITLE
Increased backdrop z-index

### LINKED ISSUE #994: Sidebar content sits above overlay
The content in pull requests and issue sidebars sits above the dropdown overlay. In the image below you can see that the subscribe buttons, labels, and dividers sit above the overlay. This is also the case for assignees and milestones.

image

### PR DESCRIPTION
**Pull request by tom-sherman**
This fixes an issue where sidebar content on issues and PRs were displaying above the backdrop modal.

Closes #994

### CODE REVIEW

**Comment by tom-sherman** [comment]
@bfred-it Silly me 🤦 I've gone ahead and changed it

**Comment by bfred-it** [comment]
Can we find out when the z-index is ever useful there? Perhaps we can _demote_ them like the the `pagehead-actions`

**Comment by tom-sherman** [comment]
I'm not entirely sure what you mean by "demote", but using z-index seems to be the cleanest and most reliable solution here.

Setting `z-index: auto` on `.discussion-sidebar` gives us this weird behaviour when trying to add a label:

image

This rule has no effect on the `#partial-discussion-sidebar` div or when applied to each item like so:

```css
.discussion-sidebar-item {
    z-index: auto;
}
```

**Comment by bfred-it** [comment]
> I'm not entirely sure what you mean by "demote"

I was referring to this line of code, meaning that they probably don't need to have `position:relative`/`z-index`

https://github.com/sindresorhus/refined-github/blob/7fd3c553f3acb8d9f7b91e73acf21b8738f54e82/source/content.css#L696

However... I just saw what's happening.

1. The whole sidebar is above the backdrop, but only the buttons are visible because they have  a white background, for whatever reason.
2. The sidebar is above the backdrop because it's `position:sticky`, meaning that elements inside it can't be above the backdrop (e.g. the labels selector modal) without making the whole sidebar above the backdrop (because position:sticky/relative/absolute will create a new stacking context)

In short, I think this is literally unfixable. The only thing you can do is remove the white background so it's less visible.

**Comment by bfred-it** [comment]
> The only thing you can do is remove the white background so it's less visible.

The white background is no longer visible after #996, so that's no longer necessary:

**Comment by bfred-it** [comment]
Closing this. If you come up with another solution please send a new PR

**Comment by tom-sherman** [comment]
@bfred-it You found bugs with the solution in the pull request? Decreasing the z-index of the sidebar content by 1 fixes the issue and doesn't introduce any bugs for me

**Comment by bfred-it** [comment]
With `z-index: 19` on the sidebar, dropdowns are not clickable:

screen shot
